packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -171,14 +316,10 @@ abstract class ResidentRunner {
     }
   }
 
-  Future<String> _debugRotatePlatform() async {
+  Future<Null> _debugTogglePlatform() async {
     await refreshViews();
-    switch (await currentView.uiIsolate.flutterPlatformOverride()) {
-      case 'iOS':
-        return await currentView.uiIsolate.flutterPlatformOverride('android');
-      case 'android':
-      default:
-        return await currentView.uiIsolate.flutterPlatformOverride('iOS');
+    for (RunningDevice device in runningDevices) {
+      await device.togglePlatform();
```

Comment:
it seems like it would make more sense to make all the devices switch to the same platform. Maybe base this on the first device's current platform?